develop/dev-guide-transaction-restraints.md

```diff
@@ -713,7 +713,7 @@ mysql> SELECT * FROM T2;
 基本原则是要限制事务的大小。TiDB 对单个事务的大小有限制，这层限制是在 KV 层面。反映在 SQL 层面的话，简单来说一行数据会映射为一个 KV entry，每多一个索引，也会增加一个 KV entry。所以这个限制反映在 SQL 层面是：
 
 - 最大单行记录容量为 120MB（TiDB v5.0 及更高的版本可通过 tidb-server 配置项 `performance.txn-entry-size-limit` 调整，低于 TiDB v5.0 的版本支持的单行容量为 6MB）。
-- 支持的最大单个事务容量为 10GB（TiDB v4.0 及更高版本可通过 tidb-server 配置项 `performance.txn-total-size-limit` 调整，低于 TiDB v4.0 的版本支持的最大单个事务容量为 100MB）。
+- 支持的单个事务容量默认为 100 MB，最大为 1 TB（TiDB v4.0 及更高版本可通过 tidb-server 配置项 `performance.txn-total-size-limit` 调整，低于 TiDB v4.0 的版本支持的最大单个事务容量为 100MB）。
```

Comment:
@ekexium Seems it can be 100TB if it's left default.

https://github.com/pingcap/tidb/blob/0d9a4ce8b8c2a8cebbd4737a77a8307d8134946b/cmd/tidb-server/main.go#L752-L754

Reply:
membuffer 的内存限制这里有两种机制。如果这个`txn-total-size-limit`是默认值（100MB），那就是新机制，由 memory tracker 统一控制。如果修改成不是默认值，那就是旧的机制，memdb key + value 的最大 total size 就由这个配置/变量控制。

这个配置现在应该已经 deprecate，换成同名系统变量了。

Reply:
```suggestion
- 事务的大小限制在不同版本受到不同参数的影响。
  - 低于 TiDB v4.0 的版本支持的最大单个事务容量为 100MB。
  - TiDB v4.0 及更高版本中，可通过 tidb-server 配置项 `performance.txn-total-size-limit` 调整，默认值为 100MB，最大限制为 1TB。
  - TiDB v6.5 及更高版本中，支持的单个事务容量默认受到 [memory control](/configure-memory-usage.md) 的控制，若手动设置 `performance.txn-total-size-limit` ，可以控制事务大小最大为 1 TB。
```

有点复杂，这样修改如何？

Reply:
或者直接 link https://docs.pingcap.com/zh/tidb/dev/tidb-configuration-file#txn-total-size-limit 
（记错了，并没有换成系统变量）

Reply:
LGTM, @Icemap 觉得怎么样

Reply:
我把这个链接 [tidb-configuration-file#txn-total-size-limit](https://docs.pingcap.com/zh/tidb/dev/tidb-configuration-file#txn-total-size-limit) 也加上去。Rest LGTM.

Reply:
英文的也同步更新了：https://github.com/pingcap/docs/pull/15247